linux_os/guide/services/cron_and_at/restrict_at_cron_users/file_groupowner_cron_allow/rule.yml

```diff
@@ -37,14 +43,18 @@ references:
     stigid@ol7: OL07-00-021120
     stigid@rhel7: RHEL-07-021120
 
-ocil_clause: '{{{ ocil_clause_file_group_owner(file="/etc/cron.allow", group="root") }}}'
+ocil_clause: '{{{ ocil_clause_file_group_owner(file="/etc/cron.allow", group=target_group) }}}'
 
 ocil: |-
-    {{{ ocil_file_group_owner(file="/etc/cron.allow", group="root") }}}
+    {{{ ocil_file_group_owner(file="/etc/cron.allow", group=target_group) }}}
 
 template:
     name: file_groupowner
     vars:
         filepath: /etc/cron.allow
         missing_file_pass: 'true'
+{{% if "ubuntu" in product %}}
+        gid_or_name: '112'
```

Comment:
could you please try with the group name instead and see if it works ok, I prefer it instead of  a hardcoded gid

Reply:
you are also mixing PRs together, please work on separate branches

Reply:
Yes, sorry for that :( I did not notice about an error while switching on a on separate branches.

Reply:
> could you please try with the group name instead and see if it works ok, I prefer it instead of a hardcoded gid

Done. It works also using  the group name.